meteor/server/api/userActions.ts

```diff
@@ -54,9 +55,8 @@ export function setMinimumTakeSpan (span: number) {
 */
 
 // TODO - these use the rundownSyncFunction earlier, to ensure there arent differences when we get to the syncFunction?
-export function take (rundownPlaylistId: RundownPlaylistId): ClientAPI.ClientResponse<void> {
+export const take = syncFunction((rundownPlaylistId: RundownPlaylistId): ClientAPI.ClientResponse<void> => {
```

Comment:
A minor note;
This will create an anonymous function inside the syncFunction. By naming the function it'll be slightly easier to read call-stacks when troubleshooting:

`export const take = syncFunction(function take(rundownPlaylistId: RundownPlaylistId): ClientAPI.ClientResponse<void> {`

Reply:
Good tip!